### PR TITLE
build: Bump contracts to 0.0.28 and resolve build issues

### PR DESCRIPTION
**Pull request by nadavhollander**
This PR introduces the following changes:

- Bump `@dharmaprotocol/contracts` to 0.0.28
- Resolve various build issues throwing typescript errors.

### CODE REVIEW

File: src/wrappers/contract_wrappers/debt_token_wrapper.ts [hunk 1]
```diff
@@ -560,7 +560,7 @@ export class DebtTokenContract extends BaseContract {
             return abiEncodedTransactionData;
         },
     };
-    public safeTransferFrom = {
+    public safeTransferFromWithData = {
```
Comment (saturnial) [inline src/wrappers/contract_wrappers/debt_token_wrapper.ts]: we deleted this out on the debt token api feature branch and just specified a default param for data.

Comment (nadavhollander) [reply src/wrappers/contract_wrappers/debt_token_wrapper.ts]: KK -- recreating.

File: src/apis/adapters_api.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ import {
 import {
     SimpleInterestTermsContractContract,
     CollateralizedSimpleInterestTermsContractContract,
-} from "src/wrappers";
+} from "../wrappers";
```
Comment (saturnial) [inline src/apis/adapters_api.ts]: RIP absolute paths?

Comment (nadavhollander) [reply src/apis/adapters_api.ts]: Can't seem to make these work in the build process.   I've gone deep down this rabbit hole and don't want to do it again 😬

**Pull request merged**